glob.js

```diff
@@ -123,6 +123,10 @@ function Glob (pattern, options, cb) {
     options = null
   }
 
+  if (options && options.fs) {
+    fs = options.fs
```

Comment:
this will override it permanently.

```js
glob('*.txt', { cwd: '/some/directory', fs: vol }, function (e, f) {...})

glob('*.txt', { cwd: '/some/directory' }, function (e, f) {...})
```

👆 the second call here will use the virtual file system, even though it should use the real one.